jaxlib build error in PacketMathAVX512.h

Dear jax team,

I am running into weird compilation errors when building jaxlib with GPU support. They all come from `PacketMathAVX512.h` and look like

```
external/org_tensorflow/third_party/eigen3/unsupported/Eigen/CXX11/src/FixedPoint/PacketMathAVX512.h: In function 'typename Eigen::internal::unpacket_traits<T>::type Eigen::internal::predux_min(const Packet&) [with Packet = Eigen::internal::Packet16q32i; typename Eigen::internal::unpacket_traits<T>::type = Eigen::QInt32]':
external/org_tensorflow/third_party/eigen3/unsupported/Eigen/CXX11/src/FixedPoint/PacketMathAVX512.h:432:16: error: could not convert 'Eigen::internal::pfirst<__vector(2) long long int>(_mm_min_epi32(res.Eigen::internal::eigen_packet_wrapper<__vector(2) long long int, 0>::operator __vector(2) long long int&(), _mm_shuffle_epi32(res.Eigen::internal::eigen_packet_wrapper<__vector(2) long long int, 0>::operator __vector(2) long long int&(), ((((0 << 6) | (0 << 4)) | (0 << 2)) | 1))))' from 'Eigen::internal::unpacket_traits<__vector(2) long long int>::type {aka __vector(2) long long int}' to 'Eigen::QInt32'
   return pfirst(
          ~~~~~~^
       _mm_min_epi32(res, _mm_shuffle_epi32(res, _MM_SHUFFLE(0, 0, 0, 1))));
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Full log is in https://gist.github.com/clemisch/3fd1ee08ee8f7c2aafc40e0a34f81a7e.

I am building on a machine with 2x RTX Titan and 2x GTX 1080Ti and CUDA 10.0. The filename reminds me of a recent issue #2906. I am building from github master (cleared bazel cache) and don't have any problems on other machines running GTX 1080Ti only, with similar CUDA setup. 

Sorry for this being so vague, but I don't really know where to look to investigate this error.

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -28,10 +28,10 @@ http_archive(
 #    and update the sha256 with the result.
 http_archive(
     name = "org_tensorflow",
-    sha256 = "cf78f827faee54a3993376d97f267eb615b74d5ae56f7c520c53377c2a241754",
-    strip_prefix = "tensorflow-fadfcdf27ba3973525b7ef24f5d2cb10a13231bd",
+    sha256 = "77338daa0112a62989e610fa1ee5a24afac5d139bd5bde476d6f2a82922a425e",
+    strip_prefix = "tensorflow-b99e88eee6a854f0a0e6012f01de87ae0248f8af",
     urls = [
-        "https://github.com/tensorflow/tensorflow/archive/fadfcdf27ba3973525b7ef24f5d2cb10a13231bd.tar.gz",
+        "https://github.com/tensorflow/tensorflow/archive/b99e88eee6a854f0a0e6012f01de87ae0248f8af.tar.gz",
     ],
 )
 

